### DISCUSSION TITLE
#69 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by yanyanho** [tog-ETH-er/togETHer#54] [General]
Agenda:

Community Updates
Fireside Chat
会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会：https://meet.google.com/bed-mnwi-ddr

**Comment by yanyanho** [comment]
# Dapp-Learning

## repo update

5个PR #983 ～ #987

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了2场分享会
 
- 《密码学系列课程9: zkSnark--Plonk证明系统-p1》by lyndell
   - Youtube: https://www.youtube.com/watch?v=qgzSKi6smII
   - Bilibili: https://www.bilibili.com/video/BV1M14y1z7S8/

- 《WalletConnect 技术分享》by ARC
   - Youtube: https://www.youtube.com/watch?v=fJf4sEGwm0w 
  - Bilibili: https://www.bilibili.com/video/BV1fu4y1i7ws/

**Comment by luckyyang** [comment]
Rebase Community 
 
- web3 极客日报: 
  - 最近 14 期 http://daily.rebase.network/ 
  - 日报小编和内容贡献者招募 https://mp.weixin.qq.com/s/px5Jc-EtpLLHkJl7dtp2EA
- 招聘: 2 个团队发布 4 个招聘 https://github.com/rebase-network/who-is-hiring/issues 
- 武汉 hacker house 结束 https://mp.weixin.qq.com/s/jg3LrR5GaKpFWNFanhvOKQ
  - 8 个项目
  - 16 个 hacker
